### PR TITLE
Fixes trashbag alt-click behavior

### LINKED ISSUE #10464: [Bug]: Alt Clicking trashbags still opens them
### Describe the bug

Alt-clicking trash bags allows users to open the bag. 

### What did you expect?

I expected trash bags to be un-openable, as per #9642

### Reproducible steps

Alt click a trashbag.

### PR DESCRIPTION
**Pull request by Rukofamicom**
## About The Pull Request

Closes #10464 
Alt-clicking a storage item now respects whether or not the item can be opened
Slightly improves the location of the check for normal clicking in the code 

## Why It's Good For The Game

Bugs and exploits are bad

## Testing Photographs and Procedure

image

## Changelog
:cl:
fix: Alt-clicking storage items now respects whether those items can be opened.
code: Slight code improvement on interception of left clicking storage items that can't be opened
/:cl:

### CODE REVIEW

**Comment by burn1cycle** [comment]
Is it possible your change could cause conflicts with other code for keybinds? https://github.com/BeeStation/BeeStation-Hornet/issues/10467

**Comment by Rukofamicom** [comment]
No and this isn't actively in the game yet either so it is not the cause of your issue

File: code/datums/keybinding/human.dm [hunk 1]
```diff
@@ -56,6 +56,9 @@
 	if(!equipped_belt.contents.len) // nothing to take out
 		to_chat(user, "<span class='notice'>There's nothing in your belt to take out.</span>")
 		return TRUE
+	var/datum/component/storage/STR = equipped_belt.GetComponent(/datum/component/storage)
+	if(!STR.can_be_opened)
```
Comment (itsmeow) [inline code/datums/keybinding/human.dm:60]: GetComponent bad. Add a signal `COMSIG_CAN_OPEN_STORAGE)`

**Review by itsmeow — CHANGES_REQUESTED** [review]

**Comment by Rukofamicom** [comment]
100% cba

**Comment by Rukofamicom** [comment]
Actually this bug was of my own making in the first place. 

This is not a bug with Beecode and I'm not sure exactly how I mixed myself up into thinking it was - I caused this locally.

**Pull request closed without merging**